csun-metalab/nomi
# NAM-556 -- UX: When no classes appear, remove message about reporting an error. This just makes us look bad

# Description
Removed the report bug message on empty courses.
  
# Testing
1. Login as `matthew.fritz` .
2. Visit an empty course and ensure the report bug message is gone.
